floppy/bitvisessh.bat

```diff
@@ -17,9 +17,6 @@ pushd "%BITVISE_DIR%"
 
 if exist "%SystemRoot%\_download.cmd" (
   call "%SystemRoot%\_download.cmd" "%BITVISE_URL%" "%BITVISE_PATH%"
-) else (
```

Comment:
These conditionals are used to detect if there's some kind of issue with the existence of the download script. If you're going to remove the "else" case, you might as well remove the whole conditional.

Ideally, though, you could emit an error and terminate the script if the download.cmd script is not found. However, either way is fine.